uk.co.compucorp.civicrm.hrcore/templates/CRM/HRCore/Page/UserMenu.tpl

```diff
@@ -1,28 +1,30 @@
-<div class="chr_header__user-menu">
-  <div class="chr_header__user-menu__data">
-    <span class="chr_header__user-menu__name">{$username}</span>
+<div class="chr_user-menu">
```

Comment:
1.Why is this changed? Cant we use previous values? 
2. And if we are changing the name, we should remove any existing styles for those classes.  Could you check please.

Reply:
1. it's a more generic name, that fits both the SSP and the CiviHR admin (the same component will be used in both places). `chr_header` is a component in the SSP, but it doesn't exist in the admin, hence the removal
2. As I wrote in the PR description, I will be removing the styles referring the old class name in a future PR
    > The original styles in SSP will be deleted in a future PR